Source/evac.f90

```diff
@@ -391,6 +391,7 @@ MODULE EVAC
        FC_DAMPING, EVAC_DT_MIN, V_MAX, V_ANGULAR_MAX, V_ANGULAR, &
        SMOKE_MIN_SPEED, SMOKE_MIN_SPEED_FACTOR, SMOKE_MIN_SPEED_VISIBILITY, TAU_CHANGE_DOOR, &
        HUMAN_SMOKE_HEIGHT, TAU_CHANGE_V0, THETA_SECTOR, CONST_DF, FAC_DF, &
+	   TPRE_TAU, TPRE_TAU_INER, &
```

Comment:
Can you describe the new parameters?

Reply:
https://github.com/tkorhon1/fds/pull/4 This is also based on a disscusion on google list. 
Let us wait for Timo's response also.  Please give Timo some time to respond.  
Could you please help to check the unit in data.f90?  That is an issue unsolved.  Many thanks.

Reply:
@godisreal You [wrote](https://github.com/tkorhon1/fds/pull/4#issue-261534835): 
> With TPRE_TAU and TPRE_TAU_INER agents can move around during tpre time period so that they will not block others' way. 

What do you mean with _move around_? Can you show the proposed change in a testcase and compare it with the previous solution?